CONTRIBUTING.rst

```diff
@@ -41,6 +41,24 @@ How to Contribute to the Anaconda Installer (the short version)
 
 If you want to contribute a change to both the upstream and RHEL Anaconda then follow both a) and b) separately.
 
+Finding Bugs to Fix
+-------------------
+
+The development team can mark bugs with specific keywords to show that they belong to a specific
+category. You can quickly list these by searching the Red Hat bugzilla for bugs in the
+``anaconda`` component with specific keywords in Whiteboard:
+
+- For good first issues and simple fixes, the keyword is `EasyFix <https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&classification=Fedora&component=anaconda&f1=status_whiteboard&list_id=11496717&o1=substring&product=Fedora&query_format=advanced&v1=EasyFix>`_.
+
+- For BTRFS-related issues, use keyword `Btrfs <https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&classification=Fedora&component=anaconda&f1=status_whiteboard&list_id=11496717&o1=substring&product=Fedora&query_format=advanced&v1=Btrfs>`_.
```

Comment:
Shouldn't  it be `Btrfs-related`?

Reply:
I think Btrfs should be enough in the search but I don't mind any of those.

Reply:
I still see `BTRFS`, but the correct spelling is `Btrfs`.

Reply:
Ha, I finally get what you wanted. Will fix.